win32/tests/file_win32_ut/file_win32_ut.c

```diff
@@ -32,7 +32,8 @@ void real_free(void* ptr)
 #define ENABLE_MOCKS
 
 #include "azure_c_pal/execution_engine.h"
-#include "azure_c_pal/gballoc.h"
+#include "azure_c_pal/gballoc_hl.h"
+#include "azure_c_pal/gballoc_hl_redirect.h"

```

Comment:
I like controlling the redirect via an include.

Reply:
Thanks. Was really useful upstream :)

---
In reply to: [462447967](https://github.com/Azure/azure-c-pal/pull/21#discussion_r462447967) [](ancestors = 462447967)